.depcheckrc.yml

```diff
@@ -2,3 +2,4 @@ ignores:
   # Include a reason why each package is listed
   - typedoc-plugin-external-module-map # Used by typedoc in `npm run doc`
   - typedoc-plugin-nojekyll # Used by typedoc in `npm run doc`
+  - node-fetch # We only use @types/node-fetch, not node-fetch itself
```

Comment:
If you're not using `node-fetch`, you may want to use the types for the fetch package you actually use (iirc `make-fetch-happen`, but maybe `minipass-fetch`).

Reply:
So currently I only have `"@types/node-fetch": "^2.5.12",` as a dev dependency and not `node-fetch` since I only want the type definitions for fetch options and not the functionality of `node-fetch`. Would it be best to use `make-fetch-happen`/`minipass-fetch` instead or just the types (`@types/make-fetch-happen)?